.docs/source/designsafe.apps.api.tasks.reindexing.rst

```diff
@@ -0,0 +1,42 @@
+Reindexing 
```

Comment:
This documentation needs to be as a docstring in the [task](https://github.com/DesignSafe-CI/portal/blob/master/designsafe/apps/api/tasks.py#L23)